rwengine/src/engine/SaveGame.cpp

```diff
@@ -539,7 +539,11 @@ bool SaveGame::loadGame(GameState& state, const std::string& file)
 	auto icv = iconv_open("UTF-8", "UTF-16");
 	char* saveName = (char*)state.basic.saveName;
 
+#if defined(RW_NETBSD)
+	iconv(icv, (const char**)&saveName, &bytes, &outCur, &outSize);
```

Comment:
I'm not too knowledgeable about C++, but `const_cast` would cast away the const'ness.
I think this would be a case for `static_cast`. It also works as `dynamic_cast`.
- However, don't change the code, it's just a nit and we'll hopefully replace this code soon anyway.

Also there are plenty of C style casts all over the codebase which can be cleaned up properly another time.

---

A comment / link to the manpage for the NetBSD difference would have been nice btw. However, same reason as above: don't change it, we'll replace that code with some other lib probably.